packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx

```diff
@@ -108,7 +111,8 @@ export const CoinjoinSetupStrategies = ({ account }: CoinjoinSetupStrategiesProp
     const isCustom = strategy === 'custom';
     const allChecked = connectedConfirmed && termsConfirmed;
     const allAnonymized = notAnonymized === '0';
-    const isDisabled = !allChecked || allAnonymized;
+    const isDisabled = !allChecked || allAnonymized || isTorDisabled;
+
     const buttonTooltipMessage = allAnonymized
```

Comment:
This could also be modified to include the correct reason for being disabled –Tor

Reply:
Yes! Done 62472460c7a8edfbeccd033c87e543773f40c9dd